tools/bcdb_mock/pkg/workloads/reservation.go

```diff
@@ -126,6 +126,16 @@ func (a *API) get(r *http.Request) (interface{}, mw.Response) {
 
 func (a *API) list(r *http.Request) (interface{}, mw.Response) {
 	var filter types.ReservationFilter
+	sCustomerTid := r.URL.Query().Get("customer_tid")
```

Comment:
You can add a new type to abstract the parsing of all of this, like we did here: https://github.com/threefoldtech/zos/blob/master/tools/bcdb_mock/pkg/directory/nodes_store.go#L29